crates/typst/src/foundations/calc.rs

```diff
@@ -908,6 +909,31 @@ pub fn quo(
     Ok(floor(dividend.apply2(divisor.v, Div::div, Div::div)))
 }
 
+/// Calculates the euclidean norm of a sequence of numbers.
+///
+/// ```example
+/// #calc.norm(1, 2, -3, 0.5)
```

Comment:
Would it be a good idea to either show that this norm is the Euclidian norm, either parametrize the `2` in the formula so we can compute other norms?

Raw ideas:
# calc.euclidian_norm(...) = calc.norm2(...) = calc.pnorm(2, ...)